tests/suites/test_suite_psa_crypto.function

```diff
@@ -8900,7 +8900,9 @@ void derive_input(int alg_arg,
     psa_set_key_usage_flags(&attributes, PSA_KEY_USAGE_DERIVE);
     psa_set_key_algorithm(&attributes, alg);
 
-    PSA_ASSERT(psa_key_derivation_setup(&operation, alg));
+    if (alg != 0) {
```

Comment:
Minor: the code has `0` and the test data has `PSA_ALG_NONE`. The value of `PSA_ALG_NONE` won't ever change, but it's a bit confusing for readers who aren't very familiar with the API.